libsrc/meshing/meshclass.cpp

```diff
@@ -7110,6 +7110,22 @@ namespace netgen
     else
       return defaultstring;
   }
+  const std::vector<int> & Mesh :: GetBCIDs(string name) const{
+	static std::vector<int> IDs;
+	IDs.clear();
+	if ( !bcnames.Size() ){
+		IDs.push_back(0);
+		return IDs;
+	}
+	for (int i=0; i < bcnames.Size(); i++){
+		if (bcnames[i]){
+		  if (string(*bcnames[i]) == name){
+		     IDs.push_back(i+1);
+		  }
+		}
+	}
+	return IDs;
+  }
```

Comment:
This can easily be done in python as well, is there a reason to not do it there/ make the interface more complex here?